tests/test_specs.py

```diff
@@ -39,6 +42,10 @@ def test_iana_specs_synchronized(registry_file_name: str) -> None:
 @pytest.mark.parametrize("spec", glob.glob("*.rflx"))
 def test_validate_spec(spec: str) -> None:
     pyrflx = PyRFLX.from_specs([spec], True)
+    if Path(f"tests/{spec.replace('.rflx', '')}_checksum.py").exists():
+        checksum_module = importlib.import_module(f"tests.{spec.replace('.rflx', '')}_checksum")
+        all_checksum_functions = checksum_module.checksum_functions  # type: ignore[attr-defined]
+        pyrflx.set_checksum_functions(all_checksum_functions)
```

Comment:
If I understood correctly, you want one `checksum.py` containing all checksum functions (and one message -> checksum mapping) and this module should not be in the tests directory (so that there is only one checksum module in the whole repo)? 
If so, this will not work in general because when using the cli, the name of the checksum module must be provided and this module must contain exactly one dict mapping the message names and fields to the checksum function.
The checksum dict is than passed to PyRFLX.set_checksum_functions(). If the passed dict contains message names that are not part of the pyrflx object, an error is raised. So, if we only have one checksum.py module, the checksum dict contains all message -> field -> checksum mappings and if this module is passed to e.g. IPv4, it also contains the Ethernet checksum and thus an error will be raised. 

Because of that, when adding the set_checksum_functions() method to PyRFLX in https://github.com/Componolit/RecordFlux/pull/698 I originally suggested not to raise an error when setting a checksum for an unknown package.

Reply:
> If I understood correctly, you want one `checksum.py` containing all checksum functions (and one message -> checksum mapping) and this module should not be in the tests directory (so that there is only one checksum module in the whole repo)?

Correct.

> If so, this will not work [...]. If the passed dict contains message names that are not part of the pyrflx object, an error is raised. [...]
> 
> Because of that, when adding the set_checksum_functions() method to PyRFLX in [Componolit/RecordFlux#698](https://github.com/Componolit/RecordFlux/pull/698) I originally suggested not to raise an error when setting a checksum for an unknown package.

I was not aware of the consequences for the validation tool when suggesting raising an error in PyRFLX. It would have been helpful if you had mentioned that. I think we should change the behavior of PyRFLX then.

Reply:
Yes, I could have been more specific about the validation tool on this. At the time I added the reason for not raising an error as a commit description https://github.com/Componolit/RecordFlux/pull/698/commits/e644174f1698314b64aab2a1a8c4833e4eef720b.
I opend a PR in RecordFlux to change it.